autoload/lsp.vim

```diff
@@ -380,6 +380,18 @@ function! lsp#default_get_supported_capabilities(server_info) abort
     \   'workspace': {
     \       'applyEdit': v:true,
     \       'configuration': v:true
+    \   },
+    \   'textDocument': {
+    \       'completion': {
+    \           'completionItemKind': {
+    \              'valueSet': [1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25]
```

Comment:
Can you get the supported kinds from `omni.vim` instead of hardcoding them? That way we won't forget updating this line.

Reply:
Good idea -- how would I do that in `vimL`?

Reply:
You can add a function to `omni.vim` that returns `keys(s:kind_text_mappings)` and call that here.

Reply:
Though you may also need to convert the Strings you get to Numbers, as well. `:h map()` will be useful for that.

Reply:
You can try `map(keys(...), {idx, key -> str2nr(key)})`.

Reply:
That works, although the list is sorted wrong (`[10,11,...,1,2,...]`). Seems to make no difference, though.

Reply:
`sort(..., {lhs, rhs -> lhs - rhs})` will fix that, though I don't think it matters.

Reply:
Thanks! I'd suggest leaving it for now -- I'll keep an eye open, and if something weird happens, I now know how to fix it.